src/main/java/com/yandex/money/api/typeadapters/model/showcase/container/ContainerTypeAdapter.java

```diff
@@ -48,7 +49,10 @@
     @Override
     protected void deserialize(JsonObject src, K builder, JsonDeserializationContext context) {
         for (JsonElement item : src.getAsJsonArray(MEMBER_ITEMS)) {
-            builder.addItem(deserializeItem(item, context));
+            T deserializedItem = deserializeItem(item, context);
+            if (!(deserializedItem instanceof Undefined)) {
```

Comment:
Now it is more efficient to compare instance of deserializedItem with Undefined.INSTANCE than to use instanceof operator.